include/swift/AST/Decl.h

```diff
@@ -520,7 +520,10 @@ class alignas(1 << DeclAlignInBits) Decl {
     KnownProtocol : 8, // '8' for speed. This only needs 6.
 
     /// The number of requirements in the requirement signature.
-    NumRequirementsInSignature : 16
+		NumRequirementsInSignature : 16,
```

Comment:
Seems like this, and other changes, are indented with tabs instead of spaces, like the rest of the file?

Reply:
Yikes, good catch. My editor has gone nuts.

Reply:
(I'll clean them up shortly; for now, I need to unblock CI)

Reply:
Whitespace cleaned up in https://github.com/apple/swift/pull/17277, thank you!